README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
```

Comment:
Clarifying question: are you envisioning that liquidity requests will be recursively quoted like requests for specific amounts? I think the answer should be yes, because even if we said you shouldn't do this, connectors will do it anyway to provide the best service possible to their customers.

> it'll usually be accounts for requests and prefixes for responses, but that you should test all four cases.

This is a protocol "smell" to me. The messages should not be so ambiguous that you need to check 4 cases to figure out if the response tells you what you wanted to know.

These are the cases I can think of:

1. I want a liquidity curve to a specific account. Anything less specific than that is meaningless to me and I will need to discard it anyway. In this case, I would prefer for the connector to give me an error saying "I don't know".
2. I want a liquidity curve to a ledger. Again, if you don't know the curve all the way to that ledger, I'd probably prefer an "I don't know".
3. I want a liquidity curve to as close to a given address as possible. I'm not sure this one makes sense unless you're specifically asking a connector _not_ to do a remote/recursive quote and just want an answer from their routing table. Unless there's a price difference for remote and local quotes, I don't see why you would want this.

Based on these, I don't see a use case for the connector responding for a prefix that is something other than what I asked for. If the requester will need to discard the answer anyway because it's not what they asked for, the connector should just respond with an error instead.

Regarding the field names, I think the best options are either to split out the messages into two so they can be precise, or to make the field name something that is more general and includes both accounts and ledgers like "address" or "destination".